pkgs/development/python-modules/autopep8/default.nix

```diff
@@ -22,6 +24,19 @@ buildPythonPackage rec {
     hash = "sha256-TuaDnZqn9mRUsoDJkj9JK4ztvzl9JTwAk8nghIkZBvw=";
   };
 
+  patches = [
+    # patch to support python 3.12. To remove after updating autopep8 to version > 2.0.4
+    (fetchpatch {
+      name = "fix-python-3.12.patch";
+      url = "https://github.com/hhatto/autopep8/compare/8f12c0a..af7399d.diff";
```

Comment:
How about updating `src` and `version` to `2.0.4-unstable-2023-10-27`?

Reply:
very good point. Done !
Thanks for the suggestion